test/unit.setup.js

```diff
@@ -19,10 +19,12 @@ Object.defineProperty(global, 'strapi', {
       plugin.policy = name => plugin.policies[name];
     });
 
-    strapiInstance.service = name => {
+    const oldService = value.service;
```

Comment:
when do you need this ?

Reply:
Well, it doesn't seem needed anymore. With the default value it seems to work at least